src/PopoverMenu.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import cx from 'classnames';
+
+/**
+ * @module PopoverMenu
+ */
+class PopoverMenu extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.selectedIndex = 0;
+		this.updateFocusBy = this.updateFocusBy.bind(this);
+		this.handleKeyUp = this.handleKeyUp.bind(this);
+	}
+
+	updateFocusBy(delta) {
+		const menuItems = findDOMNode(this).querySelectorAll('.popover-menu-item');
+		const targetIndex = this.selectedIndex + delta;
+
+		if (menuItems[targetIndex] == undefined) {
+			return;
+		}
+
+		this.selectedIndex = targetIndex;
+		menuItems[targetIndex].focus();
```

Comment:
The idea was to allow for a generic content container inside a popover, which means you would be able to do either of the following later on:

```
<Popover>
     <PopoverMenu> ... items... </PopoverMenu>
</Popover>
```

```
<Popover>
     <PopoverContent> ... arbitrary content ... </PopoverContent>
</Popover>
```

Now that I'm looking at this though, These should probably just be separate components.